src/root/static/js/common.js

```diff
@@ -46,29 +46,17 @@ $(document).ready(function() {
           window.location = link.attr("href");
     });
 
-    bootbox.animate(false);
+    bootbox.setDefaults({ animate: false });
 
     $(".hydra-popover").popover({});
 
-    $(function() {
-        if (window.location.hash) {
-            $(".nav-tabs a[href='" + window.location.hash + "']").tab('show');
-        }
-
-        /* If no tab is active, show the first one. */
-        $(".nav-tabs").each(function() {
-            if ($("li.active", this).length > 0) return;
-            $("a", $(this).children("li:not(.dropdown)").first()).tab('show');
-        });
+    /* Deep emedding for tabs. */
```

Comment:
*emedding* isn't a word :).

Another way to identify this is:

```suggestion
    /* Activates tab according to URL anchor. */
```